docs/source/en/model_doc/vitpose.md

```diff
@@ -90,54 +74,7 @@ with torch.no_grad():
     outputs = model(**inputs)
 
 pose_results = image_processor.post_process_pose_estimation(outputs, boxes=[person_boxes])
```

Comment:
I notice that with this sample image, it detects various people, however I think ideally you only run ViTPose on each person individually rather than on an image containing various people...  

![image](https://github.com/user-attachments/assets/c2a12ae4-48ef-4552-98f1-8457325b0362)